bin/colcon

```diff
@@ -23,12 +23,14 @@ from colcon_core import entry_point  # noqa: E402
 custom_entry_points = {}
 
 
-def custom_load_entry_points(group_name):  # noqa: D103
+def custom_load_entry_points(group_name, *, exclude_names=None):  # noqa: D103
```

Comment:
The change looks fine, but I'm curious. Why does `bin/colcon` monkey patch `entry_point.load_entry_points`? What would happen if it used the original?

Reply:
In thi bootstrap script `colcon` has not been installed yet so there are no entry points registered. This logic provide the entry points to bootstrap `colcon`. Once `colcon` is installed this script isn't used anymore (see the comment at the top of the file).